src/parseRuntimeAndValidateSDK.ts

```diff
@@ -74,13 +75,16 @@ export function getRuntimeChoice(sourceDir: string): string {
   }
   const runtime = ENGINE_RUNTIMES[engines.node];
   if (!runtime) {
+    track("functions_runtime_notices", "package_missing_runtime");
     throw new FirebaseError(UNSUPPORTED_NODE_VERSION_MSG, { exit: 1 });
   }
 
   if (runtime === "nodejs6") {
+    track("functions_runtime_notices", "nodejs6_deploy_prohibited");
     throw new FirebaseError(UNSUPPORTED_NODE_VERSION_MSG, { exit: 1 });
   } else {
     if (functionsSDKTooOld(sourceDir, ">=2")) {
```

Comment:
This looks like its never defined?

Reply:
It's defined further down in the file, lint just doesn't like that.